data/haarcascades_cuda/haarcascade_fullbody.xml

```diff
@@ -138,7 +138,7 @@ Thanks to Martin Spengler, ETH Zurich, for providing the demo movie.
 -->
 <opencv_storage>
 <haarcascade_fullbody type_id="opencv-haar-classifier">
-  <size>14 28</size>
+  <size>28 14</size>
```

Comment:
> `14x28 fullbody detector`

That change contradicts the description.

Bodies are usually vertically aligned, so `height > width`